hub/core/dataset/dataset.py

```diff
@@ -426,19 +433,63 @@ def checkout(self, address: str, create: bool = False) -> str:
     def log(self):
         """Displays the details of all the past commits."""
         commit_node = self.version_state["commit_node"]
-        logger.info("---------------\nHub Version Log\n---------------\n")
-        logger.info(f"Current Branch: {self.version_state['branch']}")
+        print("---------------\nHub Version Log\n---------------\n")
```

Comment:
Are we comprehensively changing these emits or cherry picking as we go along?

Reply:
We only have 2 functions that deliberately print out something, diff and log.
Realized that print was a better choice while implementing diff, changed log too for consistency.

Reply:
The cluster setup (atm) will capture `stdout` emits as log. Strongly suggest removing any free form debugging logs, and strictly stick to `f"<log-level> <log-msg> - <key-0>: <val-0> - [...] <key-n>: <val-n>"` formatting of logs. For debug (check with @absinha18) it is possible that `stderr` is ignored and can be used for free-form debugging emits. We wants logs that we can systemically grep.